### PR TITLE
Helpers for seq::Addr

### PR DESCRIPTION
**Pull request by calvinkosmatka**
Hi @diwic, wanted to add a couple of helpers for working with sequencer addresses. One to get an Addr from a PortInfo struct, and a FromStr implementation for addresses in the form "[client]:[port]". Let me know what you think!

### CODE REVIEW

File: src/seq.rs
Comment (diwic) [inline src/seq.rs:374]: Shorten address to "addr" and add a comment - functions that do not represent a 1:1 relationship with an alsa-lib function should have a comment.

File: src/seq.rs
Comment (diwic) [inline src/seq.rs:493]: You don't need to collect into a Vec - just call `next()` on the iterator itself.

Also, it does not error out correctly in case you have too many or too few ':', i e, it should not panic on strings like '', '123', and '1:2:3'.

File: Cargo.toml
Comment (diwic) [inline Cargo.toml:3]: Let's skip the version changing commit for this PR - I can do a release after this PR if you like?

**Review by diwic — CHANGES_REQUESTED** [review]
Hi and thanks for the PR!

I think these are good changes in general and will be happy to merge it once you have addressed the review comments.

File: src/seq.rs
Comment (calvinkosmatka) [reply src/seq.rs]: Updated in the latest commit

File: Cargo.toml
Comment (calvinkosmatka) [reply Cargo.toml]: That makes sense to me.

File: src/seq.rs
Comment (calvinkosmatka) [reply src/seq.rs]: Shortened and commented!

**Comment by diwic** [comment]
Thanks, looks good now. Would you mind squashing it into one patch before I merge it?

**Comment by calvinkosmatka** [comment]
Done. Thanks for reviewing!

**Comment by diwic** [comment]
Thanks for the contribution!

**Pull request merged**
